shared/src/commonMain/kotlin/org/hyperskill/app/subscriptions/domain/model/SubscriptionType.kt

```diff
@@ -4,15 +4,21 @@ import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 
 @Serializable
-enum class SubscriptionType {
+enum class SubscriptionType(
+    val isProjectSelectionEnabled: Boolean = false,
+    val isProjectInfoAvailable: Boolean = true,
```

Comment:
Why do we have 2 flags for projects? In what real cases we have different values? I guess users can either have or not projects in app

Reply:
@XanderZhu 
I used 2 different flags because there were 2 different conditions for project selection and project info viewing in our code. So these flags are just boolean analogs of previous conditions.